HousingFinanceInterimApi.Tests/V1/Infrastructure/DatabaseContext/Reports/GetCashImportByDateTests.cs

```diff
@@ -33,6 +33,15 @@ public GetCashImportByDateTests(BaseContextTest baseContextTest)
         cleanups.Add(() => ClearTable.ClearTables(_context, tablesToClear));
     }
 
+    /*
+     * Given a list of cash loads and linked SSMiniTransactions
+     * When GetCashImportByDate is called with a date range
+     * Then it should return a list of cash imports by date
+     *     within the date range
+     *     with the total value of transactions for each rent group
+     *     with summed totals across all rent groups
+     */
+
     [Fact]
     public async void ShouldGetCashImportByDate()
```

Comment:
Expanded this test to insert multiple records and ensure that they were being grouped and aggregated correctly